test/solidity/Facets/HopFacet.t.sol

```diff
@@ -79,15 +77,18 @@ contract HopFacetTest is TestBaseFacet {
 
         // adjust bridgeData
         bridgeData.bridge = "hop";
-        bridgeData.destinationChainId = 137;
+        bridgeData.destinationChainId = DSTCHAIN_ID;
 
         // produce valid HopData
         validHopData = HopFacet.HopData({
             bonderFee: 0,
             amountOutMin: 0,
             deadline: block.timestamp + 60 * 20,
             destinationAmountOutMin: 0,
-            destinationDeadline: block.timestamp + 60 * 20
+            destinationDeadline: block.timestamp + 60 * 20,
+            relayer: address(0),
+            relayerFee: 0,
+            nativeFee: 0
```

Comment:
Is this really valid data (I thought a nativeFee has to be sent with every tx...so I would have expected a value here)?

Reply:
Yes, those values are currently only needed for Goerli/Linea transfers.